codalab/apps/web/tasks.py

```diff
@@ -775,9 +775,9 @@ def send_mass_email(competition_pk, body=None, subject=None, from_email=None, to
 
 @task(queue='site-worker')
 def do_phase_migrations():
-    logger.info("Task: Do phase migrations running.")
     competitions = Competition.objects.filter(is_migrating=False)
-
+    count_competitions = len(competitions)
```

Comment:
Sorry to keep nitpicking! I should have just said use `len(competitions)` in place of the variable